movies/docs/deployment.md

```diff
@@ -0,0 +1,14 @@
+## Deployment on Heroku
+
+To be able to deploy the live demo to Heroku, you need access to the Hydra-CG Heroku account and log in via:
```

Comment:
Not really important, but maybe a ["deploy to heroku" button](https://devcenter.heroku.com/articles/heroku-button) could effectively replace these instructions?

Reply:
Did not know that, but I wonder if this is really easier then runnig `make deploy` in the end. I would prefer to have a continuous delivery pipeline and make manual interaction obsolete

Reply:
The purpose of the button is for anyone else to deploy if they so wish

Reply:
Ok, but let's tackle this in another issure / PR and finish this one